content/ios/user.md

```diff
@@ -16,10 +20,13 @@ completionHandler:^(SKYUserRecordID *user, NSError *error) {
 }];
 ```
 
-TODO: It checks for uniqueness username
-TODO: Remind developer to use email as username when necessary
+This call will create a new user in your app. Before completing the process, it automatically checks if the username is unique. New users should always be created with this method.
```

Comment:
- Better state what "This call" is referring to?
- "should always be created with this method", but 3 paragraphs later there is a `signupWithEmail` method. Better move the `signupWithEmail` code to immediately follow `signupWithUsername` to make the choices clear? The following paragraphs talk about the general result after calling the signup function, so they will apply to both the cases for signing up with username or email.